ansible/ci-playbooks/roles/ci-base-setup/tasks/redhat.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: ci-base-setup (redhat) | Set hostname
+  hostname:
+    name: "{{ inventory_hostname }}"
```

Comment:
This possible just needs to be conditionally applied if, say, the hostname matches the hostname from the initial OpenStack image you were cloning from, or something like that.